libs/partners/mongodb/tests/unit_tests/test_vectorstores.py

```diff
@@ -75,6 +76,9 @@ def _validate_search(
         output = vectorstore.similarity_search("", k=1)
         assert output[0].page_content == page_content
         assert output[0].metadata.get("c") == metadata
+        # Validate the ObjectId provided is json serializable
+        assert loads(dumps(output[0].page_content)) == output[0].page_content
+        assert loads(dumps(output[0].metadata)) == output[0].metadata
```

Comment:
And then here you can show how this would be used in practice.
Notice that I'm even using bson.json_util.loads on a string dumped using plain json.dumps

```
        from bson import json_util, ObjectId
        json_metadata = dumps(output[0].metadata). # normal json.dumps 
        assert isinstance(json_util.loads(json_metadata)["_id"], ObjectId)
```

Reply:
I think most folks in the LangChain workflow would end up using either `orjson` or `json` library especially as the underlying LangChain "pipes" leverage either of those two. 

I'll add this in as I think it's still valuable.